environment.yml

```diff
@@ -26,53 +26,52 @@ dependencies:
   - cuda-nvrtc=12.3.52                  # cuda
   - cuda-nvrtc-dev=12.3.52              # cuda, dev
   - dcmtk=3.6.1
-  - deepdiff=5.8
-  - doxygen=1.9                         # dev
+  - deepdiff=7.0.1
+  - doxygen=1.10.0                          # dev
+  - eigen=3.4.0
   - fftw=3.3.9
-  - gadgetron::gadgetron-python=1.4.1
-  - gcc_linux-64=9.4.0                  # dev
-  - gdb=11.1                            # dev
-  - gmock=1.10.0                        # dev
-  - gxx_linux-64=9.4.0                  # dev
-  - gtest=1.10.0                        # dev
-  - h5py=3.2.1
+  - gadgetron-python=1.4.1
```

Comment:
How come you have removed the channel specification on these packages?

Reply:
they are not needed. there are no ambiguity for the ismrmrd and gadgetron packages.